examples/libf3d/python/tkinter/minimal_tkinter.py

```diff
@@ -0,0 +1,37 @@
+import tkinter as tk  # Works also on CustomTKinter
+
+import f3d
+from pyopengltk import OpenGLFrame
+
+class Frame(OpenGLFrame):
+    # !!!OpenGlFrame requires adding own code to initgl and redraw!!!
+    # !!!This solution only renders the F3D viewer, as it uses EXTERNAL, so controls need to be defined!!!
+    def __init__(self):
+        super().__init__()
+        self.engine = None
+
+    # Initialize F3D
+    def initgl(self):
+        self.engine = f3d.Engine(f3d.Window.Type.EXTERNAL)
+        self.engine.loader.load_geometry(
+            f3d.Mesh(
+            points=[0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 1.0, 0.0, 0.0],
+            face_sides=[3],
+            face_indices=[0, 1, 2],
+            )
+        )
+
+    def redraw(self):
+        self.engine.window.render()
+
+if __name__ == "__main__":
+    # Create main window and define size, position and title
+    root = tk.Tk()
+    root.geometry('640x480+100+100')
+    root.title('LanPtr3D')
```

Comment:
missed that one on the first pass, the old prototype title should be replaced with something related to F3D and/or the example name